website/language.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 """Various text used throughout the website, e.g. status messages, errors, etc.
 """
-
+from django.utils.translation import ugettext_lazy as _
```

Comment:
After speaking with BrianG, let's move this `/api/language.py`. That file doesn't exist, so you'll need to create it.